workspace/__ardulib__/Graphics/XGraphics.inl

```diff
@@ -0,0 +1,47 @@
+
+XGraphics::XGraphics() {
+    _parent = nullptr;
+}
+
+const XColor* XGraphics::getFGColor() const {
+    return _parent ? _parent->getFGColor() : nullptr;
+}
```

Comment:
In many places, it is expected that `XColor*` might never be `nullptr`. Intuitively, it is a sane assumption. To avoid shooting the leg, I’d change the method signature to plain `XColor` instead of `XColor*`. The default may be mono-white or mono-black then (inverse of the background default).